gm4_area_markers/pack.mcmeta

```diff
@@ -0,0 +1,24 @@
+{
+  "pack":{
+    "pack_format":0,
+    "description":"A Gamemode 4 Module"
+  },
+  "module_name":"Area Markers",
+  "module_id":"area_markers",
+  "site_description":"Set up floating messages with a simple name tag!",
+  "site_categories":[],
+  "video_link":"",
+  "wiki_link":"",
+  "required_modules":[
+    "custom_crafters"
+  ],
+  "recommended_modules":[],
+  "credits":{
+	"Creator":[
+	  ["Bunnygamers","https://twitter.com/BunnygamersMC"]
+	],
+    "Code partially copied from":[
```

Comment:
This is unnecessary, it doesn't matter if you copied my code :P

Reply:
I am trying to credit you but I will remove if you really want me to

Reply:
Go ahead and remove it